argocd/controlplanesecret.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: controlplane
+  namespace: argocd
+  labels:
+    argocd.argoproj.io/secret-type: cluster
+type: Opaque
+stringData:
+  name: controlplane
+  server: https://controlplane.vcluster.svc
+  # config: |
+  #   { "bearerToken": "", "tlsClientConfig": { "insecure": true }}
```

Comment:
forgotten, or intentionally commented out?

Reply:
Intentionally as it's not clear what we have to put in that.